docs/GETTING_STARTED.md

```diff
@@ -1,288 +1,242 @@
+also work.
 
-## Push an image to the registry using ctr
-First things first, let's choose our favorite image.  For the purpose of this
-document we will use the latest image of rabbitmq from DockerHub, aptly named
-`docker.io/library/rabbitmq:latest`.
+> You need to change `REGISTR` to your own registry.
+> The platform tag might be different depending on your machine.
 
-To do this with ctr use the following command:
-```
-ctr i pull docker.io/library/rabbitmq:latest
+```shell
+export AWS_ACCOUNT=000000000000
+export AWS_REGION=us-east-1
+export REGISTR=$AWS_ACCOUNT.dkr.ecr.$AWS_REGION.amazonaws.com
+sudo ctr i pull docker.io/library/rabbitmq:latest
+sudo ctr i tag docker.io/library/rabbitmq:latest /rabbitmq:latest
+sudo ctr i push --platform linux/amd64 $REGISTRY/rabbitmq:latest
 ```
-To confirm this image (or the image you have chosen) is in containerd's local
-content store please run:
-```
-ctr i ls
-```
-and you should see the image.  Now we need to add an image reference it so that
-it can be added to our repository.
-```
-ctr i tag docker.io/library/rabbitmq:latest localhost:5000/rabbitmq:latest
-```
-Note that the `localhost:5000` is the address of our registry, `rabbitmq` will
-be the name of the repository, and `latest` will be the tag within the
-repository.
 
-To confirm this worked once more run `ctr i ls` and see that there is now a new
-line with the tag you just created.  Also confirm that the digest, the long
-alphanumeric hash that is prefixed by "sha256:", matches the base image.
+After this step, please check your registry to confirm the image is pushed to it.
+You can go to your registry console or use your registry's CLI (e.g. for ECR, you
+can use `aws ecr describe-images --repository-name rabbitmq`).
 
-Finally, we can push to the local registry:
+## Create and push SOCI index
 
-```
-ctr i push --platform linux/amd64 --plain-http localhost:5000/rabbitmq:latest
-```
+Instead of converting the image format, soci-snapshotter uses the SOCI index
+associated with an image to implement its lazy loading. For more detail
+please see [README](../README.md#no-image-conversion).
 
-**NOTE**: the platform tag might be different depending what environment you
-are working in.
+### Create SOCI index
 
-Once again let's confirm via a curl command:
-```
-curl http://localhost:5000/v2/_catalog
-```
-We should now see that our repository has been created
-`{"repositories":["rabbitmq"]}`.  We can further inspect and see what tags were
-created.
-```
-curl http://localhost:5000/v2/rabbitmq/tags/list
-```
-We can also look at the details of the tag using:
-```
-curl http://localhost:5000/v2/rabbitmq/manifests/latest
+Let's create a SOCI index, which later will be pushed to your registry:
+
+> The `--legacy-registry` flag specifies to create SOCI index as image manifest
+> instead of artifact.
+
+```shell
+sudo soci create --legacy-registry $REGISTRY/rabbitmq:latest
```

Comment:
Note: This will most likely need to be changed as soon as #434 is in , as 1.0 manifests will be generated by default and 1.1 artifacts will require `--oci-artifact` flag.

Reply:
yeah but it's needed now :) I'll remove this if the change for #434 is merged before this PR. If this doc PR is merged first, we need to update our doc when fixing $434.

Reply:
Sounds good